README.rst

```diff
@@ -86,9 +86,9 @@ Please report unacceptable behavior to opensource@scrapinghub.com.
 Companies using Scrapy
 ======================
 
-See https://scrapy.org/companies/
+See https://scrapy.org/companies/.
 
 Commercial Support
 ==================
 
-See https://scrapy.org/support/
+See https://scrapy.org/support/.
```

Comment:
I think that's fine not to have dot in the end of URLs, because it can get tricky with highlighting - is a dot a part of URL, or a part of a sentence? In ReST is may depend on a punctuation symbol

Reply:
> As far as I know, a url cannot end in a period.

It’s uncommon, but it’s technically possible.

Have you tried rendering the documentation locally to see whether the changes break the links or Sphinx considers the periods punctuation? You can build the documentation with `tox docs`.

Reply:
If it's tecnically possible, then this is not a good idea, even if Sphinx considers them punctuation. I'll update the PR changing the order of the sentences.